app/controllers/application_controller.rb

```diff
@@ -488,7 +488,7 @@ def valid_sort_column(param, model='work')
     elsif model.to_s.downcase == 'tag'
       allowed = %w(name created_at taggings_count_cache)
     elsif model.to_s.downcase == 'collection'
```

Comment:
Is the "collection" case still used or can we remove it?

Reply:
This is still used